.github/workflows/release.yml

```diff
@@ -0,0 +1,219 @@
+name: Release
+
+on:
+  push:
+    branches: ['main']
+
+jobs:
+  preconditions:
+    runs-on: ubuntu-latest
+    outputs:
+      repo_name: ${{ steps.repo_ids.outputs.REPO_NAME }}
+      org_name: ${{ steps.repo_ids.outputs.ORG_NAME }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Check Github token
+        run: |
+          if [ -z "${{ secrets.GITHUB_TOKEN }}"]; then
+            echo "Must provide a GITHUB_TOKEN secret in order to run release workflow"
+            exit 1
+          fi
+      - name: Check Dockerhubtoken
+        run: |
+          if [ -z "${{ secrets.DSBD_DOCKERHUB_TOKEN }}"]; then
+            echo "Must provide a DSBD_DOCKERHUB_TOKEN secret in order to run release workflow"
```

Comment:
DSBD

Reply:
Mmm... made some changes, to be fair updated workflows for the first time